### PR TITLE
[MRG] Compare scorers

### PR DESCRIPTION
**Pull request by pmyers16**
PR Description
--------------

Introduces a compare module that can find the agreement between two detectors.

Closes: https://github.com/adam2392/mne-hfo/issues/22

Merge checklist
---------------

Maintainer, please confirm the following before merging:

- [ ] All comments resolved
- [ ] This is not your own PR
- [ ] All CIs are happy
- [ ] PR title starts with [MRG]
- [ ] [whats_new.rst](https://github.com/adam2392/mne-hfo/blob/master/doc/whats_new.rst) is updated
- [ ] PR description includes phrase "closes <#issue-number>"

### CODE REVIEW

**Review by adam2392 — CHANGES_REQUESTED** [review]
Can you include tests to cover lines in mne_hfo/compare.py?

**Comment by pmyers16** [comment]
> Can you include tests to cover lines in mne_hfo/compare.py?

Should be fixed

File: tests/test_compare.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import pytest
+
+from mne_hfo import create_annotations_df
+from mne_hfo.compare import compare
+from mne_hfo.detect import RMSDetector
+from numpy.testing import assert_almost_equal
+
+
+def test_compare():
```
Comment (adam2392) [inline tests/test_compare.py:9]:
```suggestion
def test_compare_detectors():
```

File: mne_hfo/compare.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
+import numpy as np
+import pandas as pd
+from sklearn.metrics import mutual_info_score, cohen_kappa_score
+
+from mne_hfo import match_detected_annotations
+
+implemented_comparisons = ["cohen-kappa", "mutual-info"]
+
+
+def compare(clf_1, clf_2, method='cohen-kappa'):
```
Comment (adam2392) [inline mne_hfo/compare.py:10]: rename to `compare_detectors`

**Review by adam2392 — APPROVED** [review]
Just rename the functions and re-update the jupyter notebook. 

FYI: for future things that don't require the "real" data, you can just add a "python file example".

**Comment by pmyers16** [comment]
> Just rename the functions and re-update the jupyter notebook.
> 
> FYI: for future things that don't require the "real" data, you can just add a "python file example".

Okay, changes made.

**Pull request merged**